File: services/hand-modeler/app/routers/health.py
"""Health check router."""

from fastapi import APIRouter

from app.models.health import HealthResponse

router = APIRouter()


@router.get("/health", response_model=HealthResponse)
async def health_check() -> HealthResponse:
    """Check API health status."""
    return HealthResponse(status="healthy", version="0.0.1")

File: packages/api-client/src/endpoints/health.ts
import { ApiClient } from '../client';

export interface HealthResponse {
  status: string;
  version: string;
}

export function createHealthApi(client: ApiClient) {
  return {
    check: async (): Promise<HealthResponse> => {
      return client.get<HealthResponse>('/health');
    },
  };
}
